app.rb

```diff
@@ -1,162 +1,145 @@
 require 'net/http'
 require 'uri'
-require './env' if File.exists?('env.rb')
 require 'json'
 require 'intercom'
 
-# create a .txt file to write to
-export_start = Time.now.utc
-File.open("export.txt", "w") {|f| f.write("Conversation Export: #{export_start}\n\n") }
-p "Export Started At: #{export_start} \n"
+class ConversationsParser
+  attr_reader :file_name, :token, :intercom
 
-# method to write to the export file
-def write_to_export(content)
-    content = content.to_s
-    File.open("export.txt", 'a+') do |f|
-      f.puts(content + "\n")
+  def initialize(file_name, token)
+    @file_name = file_name
+    @intercom = Intercom::Client.new(token: token)
+
+    File.write(file_name, "")
```

Comment:
i write the file here because i wanna make sure there is no old data in the file